assemble/bin/accumulo-cluster

```diff
@@ -304,7 +304,8 @@ function execute_command() {
   if [[ $ARG_LOCAL == 1 ]]; then
     debugAndRun ACCUMULO_CLUSTER_ARG="${servers_per_host}" "${bin}/accumulo-service" "$service" "$control_cmd" "-o" "general.process.bind.addr=$host" "$@"
   else
-    debugAndRun "$SSH" "$host" "bash -c 'ACCUMULO_CLUSTER_ARG=${servers_per_host} ${bin}/accumulo-service \"$service\" \"$control_cmd\" \"-o\" \"general.process.bind.addr=$host\" ${*@Q}'"
+    SSH=("${INITAL_SSH[@]}" "$host" "bash -c 'ACCUMULO_CLUSTER_ARG=${servers_per_host} ${bin}/accumulo-service \"$service\" \"$control_cmd\" \"-o\" \"general.process.bind.addr=$host\" $(printf "\"%q\" " "${@}")'")
+    debugAndRun "${SSH[@]}"
```

Comment:
I get that the intent here is to avoid flattening the SSH args, but we don't need to store this as a new variable. The args to the debugAndRun function are already an array. We can just avoid flattening the SSH variable and initially assign it as an array (below) rather than bother with the intermediate variable in the first place.

However, while this does fix an issue with the SSH args being flattened (which wasn't really ever a problem, since they are hard-coded in this script anyway), it doesn't change the problems with the need to quote the contents of the bash command multiple times. The eval will not see the same thing that the debug line would have printed. It will see fewer quotes. That's not important for some of the args, since many of them won't have spaces or quotes in them, but it is a problem if this is written to handle `$@` in a general way, and it's also a problem with `$bin`, since that's a path and its contents are completely unknown and could contain spaces, so it needs to be properly quoted.